### DISCUSSION TITLE
v3.0.0

### DISCUSSION THREAD

**Discussion by DTTerastar** [ESPresense/ESPresense#624] [Announcements]
## What's Changed
* Enroll using Identity Resolving Keys by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/553
* Misc fixes by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/556
* Improvements to enroll by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/560
* Upgrade to async-web-server by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/567
* Remove huzzah flavor, make all variants sensor enabled, m5atom-matrix renamed m5atom by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/527
* Svelte devices page (/devices) by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/568
* Save ordering/filters by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/570
* Ignore annoying echo dot lost iBeacons by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/571
* Watchdog for ble controller by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/572
* Minor tweak to mqtt connection logging by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/575
* Upgrade platform to v2.0.4.1 by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/573
* Add more ui, tweak memory usage, add device configs by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/578
* UI Enhancements by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/582
* Improve auto update performance by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/584
* Add options to Wifi to scan all APs (Fixes #551) by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/588
* Additional I2C improvements by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/547
* Add SGP30 I2C Co2/TVOC sensor by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/590
* Add support for SHT30 i2c sensor #591 by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/591
* Selectable LEDs and Update Button by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/595
* Add Update button (#533) by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/612
* Update from user selectable url in gui or via mqtt by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/615
* Don't check for update if no VERSION specified (just local dev) by @DTTerastar in https://github.com/ESPresense/ESPresense/pull/621

**Full Changelog**: https://github.com/ESPresense/ESPresense/compare/v2.2.9...v3.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/ESPresense/ESPresense/releases/tag/v3.0.0'>v3.0.0</a>.</em>

**Comment by onlyoneme** [comment]
I had to rollback all my autoupdated devices back to 2.2.9 because of https://github.com/ESPresense/ESPresense/issues/607...
